src/redux/gas.ts

```diff
@@ -281,6 +302,35 @@ const getPolygonGasPrices = async () => {
   return polygonGasStationPrices;
 };
 
+const getBscGasPrices = async () => {
+  const {
+    data: { result },
+  }: {
+    data: GasFeesBscGasStationData;
+  } = await bscGasStationGetGasPrices();
```

Comment:
What happens if there is no result?

Reply:
yeah nice catch, need to add better handling here and with polygon so we can track the need to prioritize the backend work

Reply:
added handling for error reporting, not for all gas related items if there is an error it never loads atm 😅 , we should revisit this after its handled via meterology(backend gas service) so we can retry etc.

Reply:
altho it will refetch every 2-3 seconds